nifi-registry-core/nifi-registry-framework/src/main/java/org/apache/nifi/registry/service/extension/StandardExtensionService.java

```diff
@@ -188,13 +188,14 @@ public ExtensionBundleVersion createExtensionBundleVersion(final String bucketId
 
             // create the bundle version in the metadata db
             final String userIdentity = NiFiUserUtils.getNiFiUserIdentity();
+            final long bundleCreatedTime = extensionBundle.getCreated().getTime();
 
             final ExtensionBundleVersionMetadata versionMetadata = new ExtensionBundleVersionMetadata();
             versionMetadata.setId(UUID.randomUUID().toString());
             versionMetadata.setExtensionBundleId(extensionBundle.getId());
             versionMetadata.setBucketId(bucketIdentifier);
             versionMetadata.setVersion(version);
-            versionMetadata.setTimestamp(System.currentTimeMillis());
+            versionMetadata.setTimestamp(bundleCreatedTime);
```

Comment:
This works great when the bundle is being created with the first version uploaded. But I noticed when subsequent versions are uploaded they are also getting tagged with the same creation time. Given that we don't know what was done in getOrCreateExtensionBundle, we might need to refactor the code above, or... it might be better to just revert this change to always use currentTime for the bundle version. I'm good with either